src/components/articles/reportArticleForm.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/articleStyles/reportArticleForm.scss';
+
+const InputFields = (props) => {
+
+	const { value, id, handleChange, issue } = props;
+
+	return (
+		<div>
```

Comment:
The enclosing div happens to have no class, meaning it is just a container. I would prefer you use `React.Fragment` for this.

Reply:
I have replaced the `div` with `React.Fragment`